### PR TITLE
Fibonacci Sequence

### PR DESCRIPTION
**Pull request by asutosh05**
## Description
fibonacci_sequence to lean git-flow

## Database schema changes
NO Database used

### Automated test cases added
NA

### Manual test cases run
- User Enter 10
-Verified that it print Fibonacci as expected for n = 2,5,10,30

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const prompt = require('prompt-sync')();
+
+// program to generate fibonacci series up to n terms
+
+// take input from the user
```
Comment (jjalan) [inline index.js]: We should avoid such comments as it is 100% clear from the line below.

The goal is to explain what code does not explain - often the why part.

Please do a quick google to read up on what makes up for good v/s not so good comments.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+node_modules
+
+.env
+
+package-lock.json
+
+logs
```
Comment (jjalan) [inline .gitignore]: New line missing

File: index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+console.log('Fibonacci Series:');
+
+for (let i = 1; i <= number; i++) {
+    console.log(n1);
+    nextTerm = n1 + n2;
+    n1 = n2;
+    n2 = nextTerm;
+}
+
+//End
```
Comment (jjalan) [inline index.js]: New line missing

**Review by jjalan — CHANGES_REQUESTED** [review]
PR should title should be friendly (as they show up in commit history)

Ex: 'fibonacci sequence'

Please be explicit about test cases. Ex: - `_To run the program run the command npm install then npm start
`

This does not tell reviewer what tests you ran.

Example.

- Verified that it prints fibnoccci sequence as expected for n = -1, 0, 1, 10 , 15,

File: index.js (hunk 1 above)
Comment (jjalan) [reply index.js]: Ex: https://itnext.io/what-makes-a-good-code-comment-5267debd2c24

Comment (jjalan) [reply index.js]: https://news.ycombinator.com/item?id=18157047
https://news.ycombinator.com/item?id=23749676
https://news.ycombinator.com/item?id=19313562

Comment (asutosh05) [reply index.js]: I went through the blogs. Explaining what does the code do is a bad comment as it does make sense.

File: .gitignore (hunk 2 above)
Comment (asutosh05) [reply .gitignore]: New line added

File: index.js (hunk 3 above)
Comment (asutosh05) [reply index.js]: New line added

**Comment by asutosh05** [comment]
Pull request name changed.
Tests ran added

**Review by jjalan — APPROVED** [review]

**Pull request merged**